src/test/java/com/easypost/Fixture.java

```diff
@@ -0,0 +1,175 @@
+package com.easypost;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Fixture {
+
+    public static int pageSize() {
+        return 5;
+    }
+
+    public static String uspsCarrierAccountID() {
+        return "ca_8dc116debcdb49b5a66a2ddee4612600";
```

Comment:
This needs to be the carrier account ID of the USPS carrier account for the Java user. This is the one for the PHP user.